### PR TITLE
LCH-4747: Exclude sites when during command execution

### PR DESCRIPTION
**Pull request by kaynenh**

### CODE REVIEW

File: src/Platform/ACSFPlatform.php [hunk 1]
```diff
@@ -268,10 +270,13 @@ public function getAcsfClient() : AcsfClient {
   public function getPlatformSites(): array {
     $sites = [];
     foreach ($this->getAcsfClient()->listSites() as $site) {
+      if (in_array($site['domain'],$this->get(self::EXCLUDED_SITES))) {
```
Comment (ankitsingh0188) [inline src/Platform/ACSFPlatform.php:273]: One space missing after **,**

Comment (ankitsingh0188) [inline src/Platform/ACSFPlatform.php:273]: One space missing after ,

**Review by ankitsingh0188 — CHANGES_REQUESTED** [review]

File: src/Platform/ACSFPlatform.php (hunk 1 above)
Comment (kaynenh) [reply src/Platform/ACSFPlatform.php]: Fixed

**Comment by kaynenh** [comment]
I'm of the opinion we should not merge this PR. This was intended as a timebox to show how we could potentially do this. I'm going to close it but it will be here to help us fully implement this.

**Pull request closed without merging**